Sources/Bob/APIs/TravisCI/TravisCI.swift

```diff
@@ -48,11 +70,13 @@ public class TravisCI {
     }
     
     private let config: Configuration
+    private let drop: Droplet
     /// Initializes the object with provided configuration
     ///
     /// - Parameter config: Configuration to use
     public init(config: Configuration) {
```

Comment:
We could add the `Droplet` to the initializer (with a default value) so that clients can provide their own?